libandroid-navigation-ui/src/main/java/com/mapbox/services/android/navigation/ui/v5/voice/NavigationInstructionPlayer.java

```diff
@@ -35,44 +40,41 @@ private void initAudioFocusRequest() {
     }
   }
 
-  private void initInstructionPlayer(Context context, @Nullable String awsPoolId,
-                                     Locale locale) {
-    if (!TextUtils.isEmpty(awsPoolId)) {
-      instructionPlayer = new PollyPlayer(context, awsPoolId, locale);
-      isPollyPlayer = true;
-    } else {
-      instructionPlayer = new DefaultPlayer(context, locale);
-    }
-    instructionPlayer.addInstructionListener(this);
+  private void initInstructionPlayers(Context context, Locale locale) {
+    mapboxSpeechPlayer = new MapboxSpeechPlayer(context, locale, this);
+    androidSpeechPlayer = new AndroidSpeechPlayer(context, locale, this);
   }
 
-  @Override
-  public void play(String instruction) {
-    instructionPlayer.play(instruction);
+  public void play(@NonNull String instruction, @Nullable VoiceInstructionMilestone voiceInstructionMilestone) {
+    instructionQueue.add(Pair.create(instruction, voiceInstructionMilestone));
+    if (voiceInstructionMilestone != null) {
+      mapboxSpeechPlayer.play(voiceInstructionMilestone.getSsmlAnnouncement());
+    } else {
+      mapboxSpeechPlayer.play(instruction, "text"); // For cases like offroute
```

Comment:
Will `textType`s be extracted into constants in MAS?  I think extracting `"text"` into a constant here can add some context when reading this code